libs/components/code-examples/src/lib/modules/inline-form/inline-form/custom-buttons/example.component.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { SkyInlineFormHarness } from '@skyux/inline-form/testing';
+
+import { InlineFormCustomButtonsExampleComponent } from './example.component';
+
+describe('Inline form custom button demo', () => {
+  async function setupTest(options: { dataSkyId?: string }): Promise<{
+    harness: SkyInlineFormHarness;
+    fixture: ComponentFixture<InlineFormCustomButtonsExampleComponent>;
+  }> {
+    await TestBed.configureTestingModule({
+      imports: [InlineFormCustomButtonsExampleComponent, NoopAnimationsModule],
+    }).compileComponents();
+
+    const fixture = TestBed.createComponent(
+      InlineFormCustomButtonsExampleComponent,
+    );
+    const loader = TestbedHarnessEnvironment.loader(fixture);
+    const harness = await loader.getHarness(
+      SkyInlineFormHarness.with({ dataSkyId: options.dataSkyId }),
```

Comment:
In the past we've been careful to only use filters if requested by the test calling the setup method. Does this work if we pass `undefined` into the filter? I'd like to keep a consistent look and feel across all of our component harness and example tests.